CHANGES

```diff
@@ -1 +1 @@
-Please see docs/source/whats_new.rst for a changelog.
+Please see docs/source/whatsnew/index.rst for a changelog.
```

Comment:
The index file doesn’t really tell you anything, so maybe point to the directory so a specific version can be selected?  Alternatively have a link to the built docs?
```suggestion
Please see docs/source/whatsnew/ for a changelog.
```

Reply:
I think pointing to the directory makes sense, so have updated that plus addressed three more broken links and a redirect.